cmd/templates.go

```diff
@@ -30,6 +30,36 @@ mirror:
 #        - name: cluster-logging
 #          channels:
 #            - name: 'stable'
+{{- if .DuProfile }}
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v{{ .Channel }}
+      packages:
+#        - name: advanced-cluster-management
+#          channels:
+#             - name: 'release-2.6'
+#             - name: 'release-2.5'
+#        - name: multicluster-engine
+#          channels:
+#             - name: 'stable-2.1'
+#             - name: 'stable-2.0'
+        - name: local-storage-operator
+          channels:
+             - name: 'stable'
+        - name: ptp-operator
+          channels:
+            - name: 'stable'
+        - name: sriov-network-operator
+          channels:
+            - name: 'stable'
+        - name: cluster-logging
+          channels:
+            - name: 'stable'
+    - catalog: registry.redhat.io/redhat/certified-operator-index:v{{ .Channel }}
+      packages:
+        - name: sriov-fec
+          channels:
+            - name: 'stable'
+{{- end }}
 {{- if eq .Channel "4.10" }}
```

Comment:
Should this section now be dropped, as it seems to collide with the .DuProfile section?

Reply:
Yes, I think we are not going to support 4.10. That was the case where some operators were removed such as the PAO. Thanks!